integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/clustertemplate/DistroXTemplateTestDto.java

```diff
@@ -44,7 +43,8 @@ public DistroXTemplateTestDto valid() {
         return withName(getResourcePropertyProvider().getName(getCloudPlatform()))
                 .withEnvironmentName(getTestContext().get(EnvironmentTestDto.class).getName())
                 .withCluster(getTestContext().init(ClusterTestDto.class).getRequest())
-                .withInstanceGroups(getTestContext().init(InstanceGroupTestDto.class).getRequest());
+                .withInstanceGroups(getTestContext().init(InstanceGroupTestDto.class).getRequest())
+                .withGatewayPort(getCloudProvider().gatewayPort(this));
```

Comment:
Wouldn't be possible to has this withGatewayPort() call in the MockCloudProvider class only? I guess other providers will work without it.

Reply:
Maybe you are right. I don't think about this yet. If the test doesn't need this, I will remove this part as well